tests/FilterRule/LowerTest.php

```diff
@@ -35,6 +35,27 @@ public function testLowerFilterRule($value, $filteredValue)
     }
 
     /**
+     * @dataProvider getLowerResults
+     * @param string $value
+     * @param string $filteredValue
+     */
+    public function testLowerFilterRuleMultiByte($value, $filteredValue)
+    {
+        $value = mb_convert_encoding($value, 'utf-16', 'utf-8');
+        $filteredValue = mb_convert_encoding($filteredValue, 'utf-16', 'utf-8');
+
+        $this->filter->setEncodingFormat('utf-8');
+
+        $this->filter->value('test')->lower();
+
+        $result = $this->filter->filter([
+            'test' => $value
+        ]);
+
+        $this->assertEquals($result['test'], $filteredValue);
```

Comment:
This test passes, even though I converted the values to UTF-16 and converted the encoding format to UTF-8. I find this feature hard to test.